src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {

```

Comment:
Don't pass expressions in the constructor. For better code readability, assign `radius` to a local variable before passing it to the constructor.